ext/drreg/drreg.c

```diff
@@ -79,6 +79,14 @@
  */
 #define AFLAGS_ALIAS_REG (DR_REG_STOP_GPR + 1)
 
+/* Used to indicate that "no offset" is being used. */
+#define NO_OFFS 0xffffffff
+
+/* Value used as base to help differentiate the offset for a TLS and mcontext
+ * spill added by non-drreg routines.
+ */
+#define NON_DRREG_TLS_SPILL_OFFS_BASE 0xf00000
```

Comment:
How is this not conflicting?  Are we assuming non-sentinel offsets are never this high?